app/views/PremiumFeaturesView.coffee

```diff
@@ -0,0 +1,17 @@
+RootView = require 'views/core/RootView'
+SubscribeModal = require 'views/core/SubscribeModal'
+template = require 'templates/premium-features-view'
+
+module.exports = class PremiumFeaturesView extends RootView
+  id: 'premium-features-view'
+  template: template
+
+  i18nData:
+    heroesCount: '12'
+    levelsCount: '330'
```

Comment:
This data should be refactored out and shared with [SubscribeModal](https://github.com/codecombat/codecombat/blob/master/app/views/core/SubscribeModal.coffee#L14-L18). Might also be good to have a task (content?) somewhere to update them from time to time.